XYZDrivers/src/main/resources/connection.default.properties

```diff
@@ -1,4 +1,4 @@
 # Connection configuration. **DO NOT** check in changes to this relating to your personal environment, or you'll break it for everyone else.
-ConnectionUrl=jdbc:derby://localhost:1527/XYZDrivers
+ConnectionUrl=jdbc:derby://localhost:1527/xyzdrivers
 ConnectionUsername=root
```

Comment:
I'm not sure this is supposed to be checked in. I know Tony did an initial commit to show an example, but there's no need to update the remote repo right? I see you've added it the git ignore, so not sure why it's been checked in.

Reply:
@lenfred123 This file is connection.default.properties, not connection.properties. Probably shouldn't check in anyway

Reply:
I thought we should have it here so if you pull the repo you have an idea for what the properties file should look like.

Reply:
Ah fair enough - in which case it makes sense to check it in it!